packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -55,24 +55,15 @@ export const Checkbox = compose<ICheckboxType>({
       [toggleChecked, userProps, state, pressable.props]
     );
 
-    let accessibilityStates: string[] = [];
-    if (state.disabled) {
-      accessibilityStates = ['disabled'];
-    } else if (state.checked) {
-      accessibilityStates = ['checked'];
-    } else {
-      accessibilityStates = ['unchecked'];
-    }
-
     const slotProps = mergeSettings<ICheckboxSlotProps>(styleProps, {
       root: {
         rest,
         ref: buttonRef,
         ...pressable.props,
         accessibilityRole: 'checkbox',
-        accessibilityLabel: ariaLabel || label,
```

Comment:
Looks like I removed `accessibilityLabel` by mistake and then added it myself. I reverted it back to the original code.